Client/Frontend/Widgets/PhotonActionSheet/TrackingProtectionMenu.swift

```diff
@@ -222,11 +222,24 @@ extension PhotonActionSheetProtocol {
 
         if items[0].count == 1 {
             // no items were blocked
-            let noblockeditems = PhotonActionSheetItem(title: Strings.TPPageMenuNoTrackersBlocked, accessory: .Text, bold: true)
-            let space = PhotonActionSheetItem(title: " ")
-
-            items = [[noblockeditems, space]]
+            var noblockeditems = PhotonActionSheetItem(title: "", accessory: .Text)
+            noblockeditems.customRender = { title, contentView in
```

Comment:
This menu has truly outgrown what it was created to do. I'm amazed at how far we've pushed it.